src/MainMenuState.cpp

```diff
@@ -54,12 +58,16 @@ void MainMenuState::HandleInput(Game &game, char input) {
 }
 
 void MainMenuState::Draw(Game &game) {
-    auto optionIterator = MainMenuState::MENU_OPTIONS.begin();
-    for (unsigned i = 0; i < MainMenuState::MENU_OPTIONS.size(); ++i, ++optionIterator) {
-        if (this->selectedOption == i) {
-            gfx::out << Attribute::STANDOUT;
-        }
-        gfx::out << Position(30, 13 + i) << std::setw(14);
-        gfx::out << optionIterator->second << gfx::nodecor;
-    }
+//    auto optionIterator = MainMenuState::MENU_OPTIONS.begin();
+//    for (unsigned i = 0; i < MainMenuState::MENU_OPTIONS.size(); ++i, ++optionIterator) {
+//        if (this->selectedOption == i) {
+//            gfx::out << Attribute::STANDOUT;
+//        }
+//        gfx::out << Position(30, 13 + i) << std::setw(14);
+//        gfx::out << optionIterator->second << gfx::nodecor;
+//    }
```

Comment:
komentarz

Reply:
w tym momencie to nic nie robi. może lepiej, żeby nic nie robiło skoro działa xD?